sprimber-cucumber/src/main/java/com/griddynamics/qa/sprimber/discovery/CucumberTestBinder.java

```diff
@@ -104,7 +99,7 @@ private void fillPreConditions(String style, Node containerNode) {
                 .forEach(containerNode::addBefore);
```

Comment:
what do you mean by addBefore? Is it ok that you have some calls with "After step" style for example?